### ISSUE TITLE
Modify array after creating a package

### ISSUE THREAD

**Issue by hgzsom** [modflowpy/flopy#122]
Is it possible to modify or replace a data array of a package (eg. KH of LPF) after creating the package object or I have to create another package object using the updated arrays?

By the way, (I am new to GitHub contributions) what are the main channel of communication for similar questions?

**Comment by jtwhite79** [comment]
yes its possible, assuming multi-layer model:
```
>>>lpf = ModflowLpf(ml,hk=2.0)
>>>lpf.hk = 1.0 #change for all layers
>>>lpf.hk[0] = 1.5 #change just layer 1
```
You can also reset the values with an array or filename of array.  Under hood, ```flopy``` is catching the attempt to reset an attribute and re-casts the new data array into the right kind of ``flopy`` array object.  

I prefer github issues...

**Comment by hgzsom** [comment]
Indeed. I used to read data with `lpf.hk.array` method and didn't realize how to modify it. 

I'm still getting to know flopy (and also some advanced features of python) but would be happy to make some contributions.

I've just looked again at the `util_3d` class. Maybe the it could also contain a `__setitem__` method for allowing value changes like
`lpf.hk[:,10,5]=1e-3` or 
`lpf.hk[mask==5]=1e-3` when using 'property masks'.

I've tried also the following without success:
`lpf.hk[:][10,5]=1e-3`
